unicef/giga-data-ingestion
# fix: typescript build errors

## What type of PR is this?

- `fix`: Commits that fix a bug

## Summary

Fixes the various typescript build errors

- Incorrect component injection pattern (`SwitcherItem` does not accept an `as` prop)
- Incorrect import
- Failed to provide a `placeholder` import. (This one is weird because the typescript definition is trying to `Pick` `placeholder` from `SelectProps `even though it doesn't exist there. It ends up being a required prop with un unknown type :gladsmad: )

![image](https://github.com/unicef/giga-data-ingestion/assets/122899250/314469c2-5b01-4648-8f3d-c1869c1fbcb4)


## How to test

1. `cd ui`
2. `npx tsc`
3. Assert no build errors
4. Check if the actual deployment if it doesnt build
